src/applications/vaos/components/PastAppointmentsList.jsx

```diff
@@ -72,6 +72,10 @@ export class PastAppointmentsList extends React.Component {
     } else if (pastStatus === FETCH_STATUS.succeeded && past?.length > 0) {
       content = (
         <>
+          <span className="vads-u-font-size--sm vads-u-font-weight--normal">
+            Showing appointments for:{' '}
+            {this.dateRangeOptions[appointments.pastSelectedIndex].label}
+          </span>
```

Comment:
We may want to add a little bit of space below this span.  Possibly with `.vads-u-display--block` and `.vads-u-margin-bottom--1`.